intro/webhooks_intro.md

```diff
@@ -141,6 +141,31 @@ Typically, you would build your webhook to respond to the Adobe challenge in a m
 
 **Note:** Make sure your response is given in the correct content type. If your webhook script places the challenge value directly in the response body, make sure it's returned as plain text (`text/plain`). For a JSON response, make sure it's `application/json`. Returning a response in the incorrect content type may cause extraneous code to be returned, which will not validate with Adobe I/O Events.
 
+#### Manual validation
```

Comment:
I think we can change the hierarchy of headings here so that developers are aware of both the validation modes. For example -

- The challenge request
    - Synchronous Validation (we need to think of a finalized name here)
       - Request
       - Response
    - Manual Validation (we need to think of a finalized name here)

@rperelma @kgera: Maybe we can introduce new terms for the previous and the new manual validation modes. For example, Azure EventGrid uses the names as `Synchronous` and `Asynchronous` validation modes. Can we reuse them? 
Another suggestion could be `Programmatic/Automated` & `Manual` validation modes.

Reply:
Yes, either option sounds good to me